### DISCUSSION TITLE
does .net 6 natively on ubuntu .does this affects fable on some positive way ? can you name it

### DISCUSSION THREAD

**Discussion by geraldodev** [fable-compiler/Fable#3112] [General]
Hi folks.

https://devblogs.microsoft.com/dotnet/dotnet-6-is-now-in-ubuntu-2204/

And if you are inclined to answer this question, I ll make another.
Does fable allows the consumption of javascript react node packages ?

How fable compares with [rscript](https://rescript-lang.org/) ?

Thanks in advance.

F# is so sexy.. not having inheritance of C languages (if i'm not mistaken)

**Comment by MangelMaxime** [comment]
Hello @geraldodev,

the benefit of .NET 6 being native to Ubuntu means that users don't need to install it. However, they still need to install Node.js so they don't have a not setup step required.

I don't think this will have much impact for users. 

Rescript and Fable 3 seems to solve similar problem offer a typed experience on top of JavaScript.

The main difference is that one created a new language for it while the other (Fable) use an existing one. One of the strength of Fable is that it emits standard JavaScript and embrace both F# and JavaScript ecosystem this helps get the most.

For example, you can use Fable with any bundler (Vite, Webpack, you name it ...), we have good/great IDE support thanks to the work done in F# etc.

Using an existing language also means that if your team already use F# over .NET or .NET you can add a new layer to your stack without having to re-learning everything. In one of my professional, project we are able to share all the domain logic between the client and server meaning that validation, types, etc. are written once and work everywhere.
